src/measure_theory/measure_space.lean

```diff
@@ -1444,8 +1436,37 @@ instance probability_measure.to_finite_measure (μ : measure α) [probability_me
 lemma probability_measure.ne_zero (μ : measure α) [probability_measure μ] : μ ≠ 0 :=
 mt measure_univ_eq_zero.2 $ by simp [measure_univ]
 
+lemma prob_add_compl_eq_one [probability_measure μ]
+  (h: measurable_set s) : μ s + μ sᶜ = 1 :=
+begin
+  rw measure_add_measure_compl,
+  rw measure_univ,
+  exact h,
+end
```

Comment:
```suggestion
by rw [measure_add_measure_compl h, measure_univ]
```
You can merge successive rewrites into just one. Also, if you provide the assumption `h` to the lemma that needs it, this becomes a one-liner.